src/pe_reports/data/data_schema.sql

```diff
@@ -40,7 +33,7 @@ CREATE TABLE IF NOT EXISTS public.domains
 CREATE TABLE public.alias
```

Comment:
```suggestion
CREATE TABLE IF NOT EXISTS public.alias
```
Add for standard on table creation